### PR TITLE
Add updated console

### PR DESCRIPTION
**Pull request by JustinPihony**
This is merely to bump the enterprise-suite components to 1.2.17

### CODE REVIEW

File: cloudflow-enterprise-components/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name: cloudflow-enterprise-components
 apiVersion: v1
-version: 2.3.1-RC0
-appVersion: 2.3.1-RC0
+version: 2.3.1
```
Comment (RayRoestenburg) [inline cloudflow-enterprise-components/Chart.yaml]: This gets set by cloudflow release.

File: cloudflow/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for installing Cloudflow
 name: cloudflow
-version: 2.3.1-RC0
-appVersion: 2.3.1-RC0
+version: 2.3.1
```
Comment (RayRoestenburg) [inline cloudflow/Chart.yaml]: Same here. No need to change it.

File: cloudflow/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+appVersion: 2.3.1
```
Comment (RayRoestenburg) [inline cloudflow/Chart.yaml]: This is not needed.

**Comment by sebastian-alfers** [comment]
@RayRoestenburg I reverted the `version` and `appVersion` changes.

**Review by RayRoestenburg — APPROVED** [review]

**Comment by sebastian-alfers** [comment]
@RayRoestenburg @JustinPihony Not sure where the conflicts come from.

**Comment by RayRoestenburg** [comment]
@sebastian-alfers because you/justin added the version, it created a new index. just remove the changes to the index.

**Comment by sebastian-alfers** [comment]
reverted the changes as well.

**Pull request merged**